src/main/resources/swagger/api-docs.yaml

```diff
@@ -1469,6 +1469,41 @@ paths:
           description: Key not found
         500:
           description: Internal server error
+  /../login:
```

Comment:
yes, this is a total hack. I'm still looking for a way to define this endpoint without changing the basePath for EVERY other endpoint.